docs/metadata/Git/git-troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Git Troubleshooting
```

Comment:
nit: are you okay with using sentence casing for titles and section headers? We have a mix of either that or title case in our docset, and I want to start enforcing one for consistency.

Git troubleshooting

Debugging Git credentials

Debugging network issues